sdk/tests/types/payload.rs

```diff
@@ -90,7 +90,9 @@ fn milestone() {
             MilestoneOptions::from_vec(vec![]).unwrap(),
         )
         .unwrap(),
-        vec![Signature::from(Ed25519Signature::new([0; 32], [0; 64]))],
+        vec![Signature::from(
+            Ed25519Signature::try_from_bytes([0; 32], [0; 64]).unwrap(),
```

Comment:
I thought this kind of stuff was not valid anymore?

Reply:
Well, all zeroes apparently works :shrug:

Reply:
Changed it